simple-store.adoc

```diff
@@ -1,27 +1,21 @@
-[[use-simple-perstistence-store]]
-Use Simple Perstistence Store
------------------------------
+[[use-simple-persistence-store]]
+= Using a Simple Persistence Store
 
 [[overview]]
-Overview
---------
+== Overview
 
-* This is a new module raincatcher-store-simple that will
-** use current functionality in the mediator ArrayStore
-** use the
+The raincatcher-store-simple module:
+* uses current functionality in the mediator ArrayStore
```

Comment:
This doesn't appear to render properly as a list:

<img width="901" alt="screen shot 2017-03-02 at 11 30 16 am" src="https://cloud.githubusercontent.com/assets/586800/23513799/b7ecdf44-ff3b-11e6-9d75-7f66c0627dfc.png">